Soccar [Client]/src/nl/soccar/library/Player.java

```diff
@@ -76,4 +76,8 @@ public void setStatistics(Statistics statistics) {
         this.statistics = statistics;
     }
     
+    @Override
+    public String toString() {
+        return getUsername();

```

Comment:
Het is netter om het field username te retourneren in plaats van de methode getUsername aan te roepen.